guides/specification.md

```diff
@@ -348,6 +352,21 @@ facts:
 
 Finally, gathered facts, are used in Check's [Expectations](#expectations) to determine whether expected conditions are met for the best practice to be adhered.
 
+## Customizable
+
+Users can modify a check's [expected values](#values) to adapt to specific system and environmental configurations.
+
+Built-in checks are considered **customizable** by **default**, so the `customizable` flag disables customization for a particular check.
+
+The customizability flag can be set global in the check and|or in [values](#customizable-values). 
+When both levels are set, the **global** flag takes **precedence**, overriding any value-level customizability.
```

Comment:
```suggestion
When customization is globally disabled for a check, that is it has been marked with `customizable: false`, it overrides any values specifics, otherwise specific values customizability is considered.
```

Reply:
I have splitted it up into two sentences. I hope this is a bit easier for the reader :)